presto-delta/src/main/java/com/facebook/presto/delta/DeltaMetadata.java

```diff
@@ -96,6 +111,55 @@ public List<String> listSchemaNames(ConnectorSession session)
         return schemas;
     }
 
+    @Override
+    public void createTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, boolean ignoreExisting)
+    {
+        PrestoTableType tableType = isExternalTable(tableMetadata.getProperties()) ? EXTERNAL_TABLE : MANAGED_TABLE;
+        Table table = prepareTable(session, tableMetadata, tableType);
+        PrincipalPrivileges principalPrivileges = buildInitialPrivilegeSet(table.getOwner());
+
+        metastore.createTable(
+                metastoreContext(session),
+                table,
+                principalPrivileges);
+    }
+
+    private static PrincipalPrivileges buildInitialPrivilegeSet(String tableOwner)
+    {
+        PrestoPrincipal owner = new PrestoPrincipal(USER, tableOwner);
+        return new PrincipalPrivileges(
+                ImmutableMultimap.<String, HivePrivilegeInfo>builder()
+                        .put(tableOwner, new HivePrivilegeInfo(HivePrivilegeInfo.HivePrivilege.SELECT, true, owner, owner))
```

Comment:
Do we support DROP table for Delta? Is DELETE permission required for that?

Reply:
Looks like DROP table support isn't there. Can you please add that too?

Reply:
DELETE permission would be required for DELETE operation on the table with SqlStandardAccessControl.
Currently, DROP Table is not supported in the connector. I will add support for dropping external table in this